act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 
-The expectations describe what the requirements are for the [=test targets=] derived from the [applicability](#applicability).
+Each expectation MUST be distinct, unambiguous, and be written in plain language. 
 
+**Note:** Atomic rules can be used for aggregation, but this isn't the primary use case. The logic for the aggregation would be described in the expectations for the rule.
 
-Expectations for Atomic Rules {#expectations-atomic}
-----------------------------------------------------
 
-An [=atomic rule=] MUST contain one or more expectations. An expectation is an assertion that is true about each [=test target=] (see [Applicability](#applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning MAY not fully quantifiable.
+### Expectations for Atomic Rules ### {#expectations-atomic}
 
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. When atomic rules are used in composite rules, the outcome of the composite rules can be `passed` even if the outcome of one of its constituent atomic rule is `failed`, depending on the [expectations](#expectations-composite) of the composite rule.
+An [=atomic rule=] expectation MUST only use information available in the [input aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures that atomic rules are encapsulated.
 
 <div class=example>
-  <p><strong>Example</strong>: A rule for labels of HTML `input` elements might have the following expectations:</p>
+  <p>**Example:** A rule for labels of HTML `input` elements might have the following expectations:</p>
   <ol>
-    <li>The test target has an accessible name (as described in [Accessible Name and Description: Computation and API Mappings 1.1](https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te)). [[accname-aam-1.1]]</li>
-    <li>The accessible name describes the purpose of the test target.</li>
+    <li>Each test target has an accessible name (as described in [Accessible Name and Description: Computation and API Mappings 1.1](https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te)). [[accname-aam-1.1]]</li>
+    <li>The accessible names describe the purpose of each test target.</li>
   </ol>
 </div>
```

Comment:
**Editorial**: We should add a note here about how to do complex aggregation in expectations, e.g. "The page has a text alternative for 80% of all img elements".

Reply:
Note: Rules can be used to do complex aggregation by describing the logic in the expectations.  e.g. - use example above.